user/user.js

```diff
@@ -3,9 +3,15 @@ const average = require('../utils/average');
 const StatusCodes = require('http-status-codes');
 
 const getUser = (req, res) => {
+  if (!req) {
+    res.sendStatus(400);
```

Comment:
Could you please use Http Status package value here

Reply:
fixed